dspace-api/src/main/java/org/dspace/content/authority/Concept.java

```diff
@@ -437,40 +437,34 @@ public static Concept find(Context context, int id) throws SQLException
     }
 
     /**
-     * Find the concept by its name - assumes name is unique
-     *
-     * @param context
-     * @param journalID
+     * Find concepts by a metadata value in the concept metadata.
      *
-     * @return the named Concept, or null if not found
+     * @return the matching Concepts, or null if not found
      */
-
-    public static ArrayList<Concept> findByJournalID(Context context, String journalID)
+    public static ArrayList<Concept> findByConceptMetadata(Context context, String searchString, String metadataSchema, String metadataElement)
```

Comment:
Can you add back a convenience method for findByJournalID, so as to minimize disruption to the public API? I think I might use that method myself in the journal-submit refactor PR.